reconcile/slack_base.py

```diff
@@ -24,3 +24,8 @@ def init_slack(slack_info, integration):
                      username=username)
 
     return slack
+
+
+def init_slack_workspace(integration):
```

Comment:
This is doing more than `init`ting. It's actually returning a slack client. I would name it accordingly

Reply:
Although I see it's a common pattern in the code 😅